Makefile

```diff
@@ -230,6 +230,7 @@ man/%.gz: man/%
 .PHONY: clean
 clean: clean_crystal ## Clean up built directories and files
 	rm -rf $(LLVM_EXT_OBJ)
+	rm -rf man/*.gz
```

Comment:
Feel free to ignore this, but maybe this line and the `rm -rf ./docs` below could be grouped into a new `clean_docs` target

Reply:
Good idea. I'd leave that for a follow-up though.